### ISSUE TITLE
All the mined blocks from my etc pool become orphan.

### ISSUE THREAD

**Issue by tommythere** [paritytech/parity-ethereum#9661]
_Before filing a new issue, please **provide the following information**._

- **Parity Ethereum version**: v1.11.11-stable  v2.0.6-stable
- **Operating system**: Linux
- **Installation**: built from source
- **Fully synchronized**: no 
- **Network**: classic
- **Restarted**:  yes

_Your issue description goes here below. Try to include **actual** vs. **expected behavior** and **steps to reproduce** the issue._

At first I am using Parity/v1.12.0-unstable as the etc node ,  the node works great for about 7 days . I have mined about 60 etc blocks.  But suddenly it comes out 5 orphan blocks continuously . I have restarted the node, and tried another parity version (v1.11.11-stable, v2.0.6-stable). But it does not work, the pool is still mining orphan blocks.   

I have checked the trace log from Parity node . My parity node is connected to 80 peers. 
The sync log is like " IO Worker #0 INFO import    80/80 peers      9 MiB chain   71 MiB db  0 bytes queue    2 MiB sync  RPC:  0 conn, 52 req/s, 167 µs"
The submiting block log is like "2018-09-14 14:33:49   INFO miner  Submitted block imported OK. #6555586: d0d1926cc4eb01f374e77bf6753cfd2304033c20b5cecf6ba43d513e451a3964"
All the logs look normal.

**Comment by 5chdn** [comment]
> 5 orphan blocks continuously

what do you mean? are all blocks ommers? or just five in a row? do you mine any accepted blocks?

**Comment by tommythere** [comment]
At the first several days ，  the blocks i mined belong to the specified miner wallet ,  that means normal.
But after that the blocks i mined , more than five in a row are all orphan .   My pool is using the following rpc for the pending block .

 curl --data '{"method":"eth_getWork","params":[],"id":1,"jsonrpc":"2.0"}' -H "Content-Type: application/json" -X POST localhost:8545

**Comment by 5chdn** [comment]
Please explain with more detail, what do you mean with "orphan"? I can't help if you don't provide a more elaborate explanation and logs.

**Comment by HaffASE** [comment]
I encountered a similar problem. I have more than 10 blocks in a row have become orphans. I used a different number of minimum peers from 25 to 200, the result is the same.
Here are my startup settings
docker run -ti -d \
-p 8545:8545 -p 30303:30303 -p 30303:30303/udp \
--name parity \
--net host \
--restart always \
-v $HOME/parity/.local:/root/.local \
parity/parity:v2.0.6 \
--mode active \
--pruning fast \
--warp-barrier 6660000 \
--no-ancient-blocks \
--no-serve-light \
--jsonrpc-interface all \
--jsonrpc-hosts all \
--min-peers 25 --max-peers 50 \
--num-verifiers 2 --jsonrpc-threads 2 \
--scale-verifiers \
--no-ws \
--no-ipc \
--db-compaction ssd \
--cache-size-db 1024 \
--tx-queue-size 64 \
--no-periodic-snapshot \
--unsafe-expose \
--relay-set strict \
--force-sealing

I recorded attach a log with 2 minutes while finding the block.

https://gist.github.com/HaffASE/d32ad8d44d2e0e77255a420aacc0bd9f

From strange, with a large number, many peers are lost (see screenshot)
https://imgur.com/a/qdzYOWM
